src/callbacks_stage/positivity_zhang_shu_dg1d.jl

```diff
@@ -35,13 +35,10 @@ function limiter_zhang_shu!(u, threshold::Real, variable,
     theta = (value_mean - threshold) / (value_mean - value_min)
     for i in eachnode(dg)
       u_node = get_node_vars(u, equations, dg, i, element)
-      set_node_vars!(u, theta * u_node + (1-theta) * u_mean,
+      set_node_vars!(u, theta * u_node + (1 - theta) * u_mean,
                      equations, dg, i, element)
     end
   end
 
   return nothing
-end
-
-
-end # @muladd
+end end # @muladd
```

Comment:
This is also a bit weird.

Reply:
This seems like a bug - maybe ask about it in JuliaFormatter?

Reply:
With current JuliaFormatter and Julia v1.9, this looks like the following:
```julia
  end

  return nothing
end
end # @muladd
```
There is no blank before the final `end`, which is not pretty either, but livable. Once could probably remedy this by adding the closing comment before the end, e.g.,
```julia
  end

  return nothing
end

# Close @muladd block
end
```